digdag-standards/src/main/java/io/digdag/standards/operator/param/RedisServerClientConnectionManager.java

```diff
@@ -0,0 +1,45 @@
+package io.digdag.standards.operator.param;
+
+import com.google.common.base.Optional;
+import io.digdag.client.config.Config;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.Transaction;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RedisServerClientConnectionManager
+        implements ParamServerClientConnectionManager
+{
+    private final String host;
+    private final Integer port;
+    private final Boolean ssl;
+    private final Optional<String> password;
+
+    public RedisServerClientConnectionManager(Config systemConfig)
+    {
+        host = systemConfig.get("param_server.host", String.class, "localhost");
+        port = systemConfig.get("param_server.port", Integer.class, 6379);
+        ssl = systemConfig.get("param_server.ssl", Boolean.class, false);
+        password = systemConfig.getOptional("param_server.password", String.class);
+    }
+
+    @Override
+    public ParamServerClientConnection getConnection()
+    {
+        Jedis redisClient = new Jedis(host, port, ssl);
+        if (password.isPresent()) {
+            redisClient.auth(password.get());
+        }
+
+        redisClient.connect();
+        return new RedisServerClientConnection(redisClient);
+    }
+
+    @Override
+    public void shutdown()
+    {
```

Comment:
The answer is `yes`, because I do not use connection pool for Redis, and Redis client connection is closed for each transaction here https://github.com/treasure-data/digdag/blob/a17e4c97e3f8303a0d22826aade3caff5744b2e3/digdag-standards/src/main/java/io/digdag/standards/operator/param/RedisParamServerClient.java#L110

But this empty implementation is confusable, so I add some comments.